crates/follows_types/src/lib.rs

```diff
@@ -2,24 +2,28 @@ use hc_link_pagination::AgentPubKeyPagination;
 use hdk::prelude::*;
 
 #[derive(Serialize, Deserialize, SerializedBytes, Clone, Debug)]
+#[serde(rename_all = "camelCase")]
```

Comment:
I'm happy to have serde serialize input types to camelCase, but it should be consistent for all the input types. Maybe for now its simplest to just keep these as the snake_case serialization and then later make another PR to convert all input types to camelCase? :smile: